packages/gatsby-tinacms-mdx/src/useMdxForm.tsx

```diff
@@ -149,8 +156,13 @@ export function useMdxForm(
     })
   }, [])
 
+  const check = (formState: any) => {
+    const error = validateMDX(formState.values.rawMdxBody)
```

Comment:
```suggestion
    const error = validateMdx(formState.values.rawMdxBody)
```

Reply:
This _should_ fix the build error too I think

Reply:
oops 🤪